backend/src/services/user.service.ts

```diff
@@ -55,10 +64,17 @@ export const updateAvatar = async (
   file: Express.Multer.File,
 ): Promise<IUser> => {
   const userRepository = getCustomRepository(UserRepository);
+  const userToUpdate = await userRepository.findById(id);
+  const fileName = userToUpdate.avatar.split('/').pop();
+  const isExistsAvatar = await isFileExists(fileName);
+
+  if (isExistsAvatar) {
+    deleteFile(userToUpdate.avatar);
```

Comment:
as i know if it will throw error then UnhandledPromiseRejectionWarning will appear which in future may terminate Node.js process

```suggestion
    await deleteFile(userToUpdate.avatar);
```